test/integration/rules/aria-hidden-focus/aria-hidden-focus.html

```diff
@@ -0,0 +1,24 @@
+<!-- Pass -->
+<p id='pass1' aria-hidden="true">Some text</p>
+<div id='pass2' aria-hidden="true"><a href="/" style="display:none">Link</a></div>
+<div id='pass3' aria-hidden="true"><button tabindex="-1">Some button</button></div>
+<input id='pass4' disabled aria-hidden="true" />
+<div id='pass5' aria-hidden="true"><a href="/" style="position:absolute; top:-999em">Link</a></div>
+
+<!-- Fail -->
+<div id='violation1' aria-hidden="true">
+	<a href="/" style="position:absolute; top:-999em">Link</a>
+</div>
+<div id='violation2' aria-hidden="true">
+	<input aria-disabled="true" />
+</div>
+<div id='violation3' aria-hidden="true">
+	<div aria-hidden="false">
+		<button tabindex="-1">Some button</button>
```

Comment:
This should pass. aria-hidden can't be reset once you've set it to true on an ancestor.